guide/src/migration.md

```diff
@@ -5,6 +5,58 @@ For a detailed list of all changes, see the [CHANGELOG](changelog.md).
 
 ## from 0.16.* to 0.17
 
+### Downcasting (`PyTryFrom`) has been changed for `PyMapping` and `PySequence` types
+
+Previously the safety checks for downcasting to `PyTryFrom` and `PyMapping`
```

Comment:
Looks the the wording in the first line might have meant to be `PyMapping` and `PySequence`? I suggest rephrasing like this:

```suggestion
### Type checks have been changed for `PyMapping` and `PySequence` types

Previously the type checks for `PyMapping` and `PySequence` (implemented in `PyTryFrom`)
```